Configure script doesn't allow change of library directories for Python 3
as it does in Python 2. Hence, hardcode our preferred paths.

Not intended for upstream.

--- openscap-1.2.9/configure.ac
+++ openscap-1.2.9/configure.ac
@@ -1070,7 +1082,7 @@ AC_ARG_ENABLE([python3],
 
 
 AC_ARG_VAR([preferred_python], [set preferred Python interpreter])
-AS_IF([test "$preferred_python" = ""], [preferred_python=$PYTHON])
+AS_IF([test "$preferred_python" = ""], [preferred_python=$PYTHON3])
 
 AC_ARG_ENABLE([perl],
      [AC_HELP_STRING([--enable-perl], [enable compilation of perl bindings (default=no)])],
@@ -1270,10 +1282,8 @@ if test "x${python3_bind}" = xyes; then
 	PYTHON3_INCLUDES=`python3-config --includes 2> /dev/null`
 
 	# the string concatenation below is just a trick to prevent substitution
-	PYTHON3_DIR=`$PYTHON3 -c "import distutils.sysconfig; \
-		print(distutils.sysconfig.get_python_lib(0,0,prefix='$' '{prefix}'))"`
-	PYTHON3_EXECDIR=`$PYTHON3 -c "import distutils.sysconfig; \
-		print(distutils.sysconfig.get_python_lib(1,0,prefix='$' '{exec_prefix}'))"`
+	PYTHON3_DIR='${prefix}/lib/python3.7/vendor-packages'
+	PYTHON3_EXECDIR='${exec_prefix}/lib/python3.7/vendor-packages'
 
 	AC_SUBST(PYTHON3_CFLAGS)
 	AC_SUBST(PYTHON3_LIBS)
@@ -1283,7 +1293,7 @@ if test "x${python3_bind}" = xyes; then
 fi
 
 # oscap-docker determine python dir on default python version
-OSCAPDOCKER_PYTHONDIR=`$preferred_python -c "import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(0,0,prefix='$' '{prefix}'))"`
+OSCAPDOCKER_PYTHONDIR="$PYTHON3_DIR"
 AC_SUBST(oscapdocker_pythondir, $OSCAPDOCKER_PYTHONDIR)
 
 
